README.md

```diff
@@ -670,6 +701,34 @@ Enable this policy on the instance or Lambda function running the Orchestrator.p
     ]
 }
 ``` 
+#### IAM Policy if CrossAccountRole
+If CrossAccountRole and CrossAccountRoleExternalId are enabled, the following IAM Policy needs to be added and attached to IAM role in target AWS account:
```

Comment:
Please BOLD the statement "...in the TARGET AWS account".

It took me a few minutes to understand what the difference was between the below policy, and the normal policy.  Upon rereading this statement, I realized this policy (and role) are for the TARGET account.

Pls update and notify me when ready for the merge

Thanks

Reply:
Statement "...in the TARGET AWS account" has been marked as bold now.